src/main/java/net/io/kino/service/AuthenticationManager.java

```diff
@@ -0,0 +1,10 @@
+package net.io.kino.service;
+
+import net.io.kino.model.User;
+
+
+public interface AuthenticationManager {
+    boolean login(User user) throws Exception;

```

Comment:
Rzucanie tak ogólniej klasy wyjątku jak Exception nie jest dobrym pomysłem, tak na prawde nie wiadomo co się stało.

Reply:
Wiesz co, miałam dokładnie to samo zdanie, ale z jakiegoś powodu metoda hash miała problemy z rzucaniem czymś innym. Ale spokojnie, to się zmieni